README.md

```diff
@@ -1,5 +1,103 @@
 # polymer-build
-A library of Gulp build tasks
 
-Note: This is an entirely different library from versions < 0.1.0 by wballard.
-The previous project is available at GitHub here: https://github.com/Custom-Elements/polymer-tools
+Polymer-build is a library for building Polymer projects. While the [Polymer CLI](https://github.com/Polymer/polymer-cli) is still the easiest way to build a project, it's customization options are limited. Polymer-build allows you to create a completely customized build pipeline, which can include additional streams and build tasks in any order. Use polymer-build over the CLI if you:
+
+- Need to customize your build(s) without using command-line flags/arguments
+- Need to run your source code through custom optimizers/processors before, after, or during build
+- Need to hook additional logic into any part of the build process
+
+
+## Installation
+
+```
+npm install --save-dev polymer-build
+```
+
+
+## Usage
+
+ The library was built for Gulp, but it can be used anywhere that Node streams can.
+
+ See the [examples/](examples/) directory for examples of how polymer-build can be used to create a full build-pipeline for your application.
+
+
+### let project = new PolymerProject(options)
```

Comment:
suggest using `const` throughout instead of `let`. I don't think you'd want someone reassigning the value of `project` to something else in their gulpfile